### ISSUE TITLE
[Data] Update VZV to consume private staging dataset locally

### ISSUE THREAD

**Issue by mddilley** [cityofaustin/atd-data-tech#13087]
Now that the VZ Socrata ETL is moved to Airflow v2, we can more easily test updates to the ETL and Socrata datasets schema. Having the test datasets open to the public isn't ideal since they may not always accurately reflect crash and people data. They could also be confused with the production datasets.

Some ideas:
- ~make the test staging datasets private~ This is done ✅
- use authentication when testing them locally (we already auth to update them, we would just have to make authenticated requests to read the data into local VZV)
- that would enable us to:
   - run the local VZ stack
   - make changes to the local DB schema
   - update the Socrata schema as needed
   - push data to the private Socrata dataset using local Airflow v2
   - make sure that test data that isn't useful to the public isn't out in the world to be confused with the production dataset

_Originally posted by @mddilley in https://github.com/cityofaustin/atd-data-tech/issues/12495#issuecomment-1635954089_

**Comment by patrickm02L** [comment]
This is the data set [Test Vision Zero Demographic Statistics - incomplete and not intended for public analysis](https://data.austintexas.gov/Transportation-and-Mobility/Test-Vision-Zero-Demographic-Statistics-incomplete/v3x4-fjgm) that we want to make private.

**Comment by chiaberry** [comment]
this should happen before LDM-ifying the socrata export

**Comment by chiaberry** [comment]
I made the staging datasets private on the data portal, ODP -> EDP

**Comment by patrickm02L** [comment]
Clicked on the link above to confirm it's private. It is. Closing

![Screen Shot 2023-10-06 at 9 18 40 AM](https://github.com/cityofaustin/atd-data-tech/assets/5566727/4d8cbca4-a628-4125-81ef-08255e77af55)

**Comment by chiaberry** [comment]
Reopening because we still need to update vzv staging to consume the private datasets i believe

**Comment by johnclary** [comment]
This was discussed again today https://austininnovation.slack.com/archives/CM9MK950S/p1720452463384849  in relation to the new Socrata export ETL https://github.com/cityofaustin/atd-vz-data/pull/1485

**Comment by johnclary** [comment]
Although this would be nice to have, I think it's clear enough on the public datasets that these are not to be used for reporting. I'm closing this because the priority is too low.
